keytool/src/main/scala/org/constellation/keytool/DeterministicKeyGen.scala

```diff
@@ -0,0 +1,43 @@
+package org.constellation.keytool
+
+import com.typesafe.scalalogging.StrictLogging
```

Comment:
We don't use StrictLogging, but an adapter that works well with cats (log4cats). See other parts of the codebase as well (Slf4jLogger)

Reply:
Just based this on prior examples, will fix.